dev/storage.h

```diff
@@ -852,11 +852,11 @@ namespace sqlite_orm {
             }
 
             template<class O, class... Cols>
-            int insert(const O& o, columns_t<Cols...> cols) {
-                static_assert(cols.count > 0, "Use insert or replace with 1 argument instead");
+            int64 insert(const O& o, columns_t<Cols...> cols) {
```

Comment:
I'd leave it like that cause changing return type of public API which exists 6+ years will spawn a lot of warnings